examples/datasets.yml

```diff
@@ -34,8 +34,10 @@ data:
       - MERCATOR_LC80210392016114LGN00_BQA.TIF
 
   - url: http://pugetsoundlidar.org/lidardata/newdata/2016/PSLC_KingCounty_Delivery1/BE_ASCII.html
-    title: "Puget Sound LiDAR Consortium"
-    func: download_puget_sound_lidar
+    title: "Puget Sound Lidar"
```

Comment:
Gulp!  Apparently the issue of how to capitalize Lidar is complicated!

![image](https://cloud.githubusercontent.com/assets/1695496/25198393/f7f13a58-250c-11e7-97be-e66d1bd4daef.png)

I guess I favor Lidar, so this change is fine.